ci/build.ps1

```diff
@@ -57,7 +57,7 @@ if ($compiler -eq 'MINGW') {
   # in MSYS2, but we cannot build inside the MSYS2 shell.
   $cmakeGenerator = 'Ninja'
   $cmakeGeneratorArgs = '-v'
-  $mingwPackages = @('ninja', 'cmake', 'perl', 'diffutils').ForEach({
+  $mingwPackages = @('ninja', 'cmake', 'diffutils').ForEach({
```

Comment:
Why is perl needed here again? It should safe to skip it if `C:/perl` in Appveyor is from http://strawberryperl.com/.

Reply:
perl is needed for git, maybe other things. Though I suppose we can try this...